modules/authentication/snip-enabling-user-authentication-with-microsoft-azure-optional-authentication-provider-steps.adoc

```diff
@@ -0,0 +1,74 @@
+:_mod-docs-content-type: SNIPPET
+
+Optional: Consider adding optional fields to the `auth.providers.microsoft` section in your `{my-app-config-file}` file:
+
+[source,yaml,subs="+quotes,+attributes"]
+----
+auth:
+  environment: production
+  providers:
+    microsoft:
+      production:
+        clientId: ${AUTHENTICATION_AZURE_CLIENT_ID}
+        clientSecret: ${AUTHENTICATION_AZURE_CLIENT_SECRET}
+        tenantId: ${AUTHENTICATION_AZURE_TENANT_ID}
+        domainHint: ${AUTHENTICATION_AZURE_TENANT_ID}
+        additionalScopes:
+           - Mail.Send
+        sessionDuration:
+          hours: 24
+        signIn:
+          resolvers:
+            - resolver: usernameMatchingUserEntityName
+              dangerouslyAllowSignInWithoutUserInCatalog: true
+signInPage: microsoft
+----
+
+`domainHint`::
+Optional for single-tenant applications.
+You can reduce login friction for users with accounts in multiple tenants by automatically filtering out accounts from other tenants.
+If you want to use this parameter for a single-tenant application, uncomment and enter the tenant ID.
+If your application registration is multi-tenant, leave this parameter blank.
```

Comment:
I'd urn this into an admonition that says that for multi-tenant app registration, the field should remain blank